cmd/vic-machine/inspect/inspect.go

```diff
@@ -112,6 +113,15 @@ func (i *Inspect) Run(cli *cli.Context) error {
 	}
 	executor.InitDiagnosticLogs(vchConfig)
 
+	installerVer := version.GetBuild()
+
+	log.Info("")
```

Comment:
Why make these separate calls instead of just doing a `fmt.Sprintf("\n...` with all of the output formatted how you want?